packages/components/inputs/rich-text-input/src/editor.styles.ts

```diff
@@ -0,0 +1,12 @@
+import styled from '@emotion/styled';
+import type { TEditorProps } from './editor';
+
+type TEditorWrapperProps = Pick<TEditorProps, 'isDisabled' | 'isReadOnly'>;
+
+const EditorWrapper = styled.div<TEditorWrapperProps>`
+  align-self: stretch;
+  cursor: ${(props) =>
+    props.isDisabled || props.isReadOnly ? 'not-allowed' : 'inherit'};
```

Comment:
In the previous version we were not using the `not-allowed` cursor for the `isReadOnly` state so I was wondering if this is an intended change.
I don't see we use it like that in other components (eg [TextInput](https://uikit.commercetools.com/?path=/story/components-inputs--textinput))

Reply:
I asked to add it here as well for consistency. https://github.com/commercetools/ui-kit/pull/2129#discussion_r814589790

Reply:
Oh! Ok. Sorry.